dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -205,6 +207,8 @@ public User createUser(String userName,
             queue = "";
         }
         user.setQueue(queue);
+        String defaultTimeZone = ZoneId.systemDefault().getRules().getOffset(Instant.now()).getId();
+        user.setTimeZone("GMT" + defaultTimeZone);
```

Comment:
Is it better to init the user timezone null? If timezone null, use the server default timezone for date convert. What do you think?